apis/v1alpha2/nginxproxy_types.go

```diff
@@ -497,6 +502,23 @@ type ContainerSpec struct {
 	VolumeMounts []corev1.VolumeMount `json:"volumeMounts,omitempty"`
 }
 
+// ReadinessProbeSpec defines the configuration for the NGINX readiness probe.
+type ReadinessProbeSpec struct {
+	// Port is the port on which the readiness endpoint is exposed.
+	// If not specified, the default port is 8081.
+	//
+	// +optional
+	// +kubebuilder:default:=8081
```

Comment:
Part of me thinks we don't actually set this default in the CRD, and just set it internally. Otherwise our default NginxProxy resource is going to render this, which I don't think it needs to since it's a niche field that people probably aren't going to update very often.